samples/python/django/README.md

```diff
@@ -0,0 +1,82 @@
+# PGAdapter and Django
+
+PGAdapter can be used in combination with Django, but with a number of limitations. This sample
+shows the command line arguments and configuration that is needed in order to use Django with
+PGAdapter.
+
+## Start PGAdapter
+You must start PGAdapter before you can run the sample. The following command shows how to start PGAdapter using the
+pre-built Docker image. See [Running PGAdapter](../../../README.md#usage) for more information on other options for how
+to run PGAdapter.
+
+```shell
+export GOOGLE_APPLICATION_CREDENTIALS=/path/to/credentials.json
+docker pull gcr.io/cloud-spanner-pg-adapter/pgadapter
+docker run \
+  -d -p 5432:5432 \
+  -v ${GOOGLE_APPLICATION_CREDENTIALS}:${GOOGLE_APPLICATION_CREDENTIALS}:ro \
+  -e GOOGLE_APPLICATION_CREDENTIALS \
+  -v /tmp:/tmp \
+  gcr.io/cloud-spanner-pg-adapter/pgadapter \
+  -p my-project -i my-instance \
+  -x
+```
+
+## Creating the Sample Data Model
+Run the following command in this directory. Replace the host, port and database name with the actual host, port and
+database name for your PGAdapter and database setup.
+
+```shell
+psql -h localhost -p 5432 -d my-database -f sample-schema.sql
+```
+
+You can also drop an existing data model using the `drop_data_model.sql` script:
+
+```shell
+psql -h localhost -p 5432 -d my-database -f drop-data-model.sql
+```
+
+## Data Types
+Cloud Spanner supports the following data types in combination with `Django`.
+
+| PostgreSQL Type                        | Django Model Field                           |
+|----------------------------------------|----------------------------------------------|
+| boolean                                | BoolField                                    |
+| bigint / int8                          | BigIntegerField                              |
+| varchar                                | CharField                                    |
+| text                                   | CharField                                    |
+| float8 / double precision              | FloatField                                   |
+| numeric(x, y)                          | DecimalField(max_digits=x, decimal_places=y) |
+| timestamptz / timestamp with time zone | DateTimeField                                |
+| bytea                                  | BinaryField                                  |
+| date                                   | DateField                                    |
+
+## Limitations
+The following limitations are currently known:
+
+| Limitation                           | Workaround                                                                                                                                         |
```

Comment:
This table should at least also include:
1. The limitation that interleaved tables are not fully supported, other than with some workarounds that come with its own limitations (including the lmitations around saving / updating rows in an interleaved table).
2. Nested transactions / savepoints.